app/src/main/java/com/jemiola/moodtimeline/views/timeline/TimelineFragment.kt

```diff
@@ -44,26 +50,78 @@ class TimelineFragment : BaseFragment(), TimelineContract.View {
     ): View? {
         if (!this::binding.isInitialized) {
             binding = FragmentTimelineBinding.inflate(inflater, container, false)
-            setupTimeline()
-            setupSearchView()
-            setupCalendarView()
+            with(binding.timelineRecyclerView) {
+                adapter = TimelineAdapter(this@TimelineFragment)
+                layoutManager = LinearLayoutManager(context)
+            }
+
+            repository = TimelineRepository()
+            val defaultToDate = repository!!.defaultSearchToDate
+            val formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy").withLocale(Locale.ENGLISH)
+            var formatDaty = defaultToDate.format(formatter)
+
+
+            binding.fromEditText.setText(formatDaty)
```

Comment:
a może tak: 
when(binding) {
  fromEditText.setText(formatDaty)
  toEditText.setText(formatDaty)
}

- [ ] change it koniecznie byq

- [ ]

Reply:
o kur, ale się walnąłem, nie when tylko with XDDdd 🤦‍♂️